docs/3. Flows/3. Flow Features/05. Integrate Google sheets in Flows.md

```diff
@@ -75,3 +75,50 @@ Click on `OK` and use the column name like below and access the value of cell va
 `@results.sheet.message_1`
 
 ![image](https://user-images.githubusercontent.com/32592458/219550825-82831f46-a239-4600-b95e-bf0533f0565b.png)
+
+
+## Write to google sheet
+
+Please find the flow created for testing purpose. The below flow is an example of the response from the user will be added directly to the google sheet. 
+
+![Screenshot1](https://github.com/glific/docs/assets/130072273/52aea06d-0693-4859-865b-ad0bdd1449a8)
+
+Below are the steps to write Google sheets in the Flows.
+
+1) Once the Flow start node is created. Then, drag the Red dot & create a new node. 
+
+![Screenshot2](https://github.com/glific/docs/assets/130072273/1bf8a56d-6c09-41f8-9f46-747d93309aef)
+
+2) Mention the **Sheet_Input** in the dropdown of **Result Name**.
+
+![Screenshot3](https://github.com/glific/docs/assets/130072273/a6680c28-8840-4386-bfb1-3a3f32fb1d76)
+
+3) Create a new node.
+
+![Screenshot4](https://github.com/glific/docs/assets/130072273/80062cd9-9a46-4617-967c-3f91756554af)
+
+4) Select **Link google sheet** from the dropdown.
+
+![Screenshot5](https://github.com/glific/docs/assets/130072273/ec032f48-3df2-40e2-be1c-e4283bf4c6f0)
+
+5) Under **Action** select **Write** from the dropdown.
+
+![Screenshot6](https://github.com/glific/docs/assets/130072273/9a29141e-8a68-49f1-91d2-ae5a19e895f5)
+
+6) Select the sheet from the dropdown.
+
+![Screenshot7](https://github.com/glific/docs/assets/130072273/f5efb017-201e-4ba0-8b3e-52ab3b89446c)
+
+7) Mention the **Sheet range** you would like to enter the data in the google sheet.
+
+![Screenshot8](https://github.com/glific/docs/assets/130072273/9964a3cb-969a-4807-ac70-ea3d94a1a50e)
+
+8) Enter the Data which needs to be written in the google sheet Under **Input data to update in sheet**.
```

Comment:
@AmitRChavan15 Add example for data that can be added, like @results.sheet_input, @contact.fields.name